schema/bom-1.5.xsd

```diff
@@ -4356,12 +4356,15 @@ limitations under the License.
     <xs:complexType name="resourceReferenceType">
         <xs:sequence>
             <xs:choice>
-                <xs:element name="ref" type="bom:refType" minOccurs="1" maxOccurs="1">
+                <xs:element name="ref" minOccurs="1" maxOccurs="1">
                     <xs:annotation>
                         <xs:documentation>
                             References an object by its bom-ref attribute
                         </xs:documentation>
                     </xs:annotation>
+                    <xs:simpleType>
+                        <xs:union memberTypes="bom:refLinkType bom:bomLinkElementType"/>
```

Comment:
is any of:
- `bom:refLinkType`: any string
- `bom:bomLinkElementType`: `anyURI` string with a certain pattern/format for BOM-Link